docs/UserGuide.adoc

```diff
@@ -741,7 +753,11 @@ After typing the command, you will remove the the team from MTM and will be take
 image::TeamRemoveExampleAfter.png[width="700"]
 // end::remove[]
 
-=== User Experience Commands
+=== Appearance Commands
+
+Here you can find useful commands to change the look of My Team Manager
+such as <<setTagColour, setting the colour of your tags>>, <<changeTeam,changing the theme>>
+and also privacy commands to hide sensitive information.
```

Comment:
I've moved up the privacy command up to fundamentals, so this shouldn't be here anymore.

Reply:
ok made the changes